### PR TITLE
fix compiler warnings introduced in d0c5ef5 and 0a395cf

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
1st commit needed for master, 2nd doesn't harm there, both needed for 2.2

@lvinken: could please glance over this?

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
@lvinken: could please glance over this?

File: mscore/exportxml.cpp [hunk 1]
```diff
@@ -2538,11 +2538,11 @@ static void writeTypeAndDots(XmlWriter& xml, const Note* const note)
       Tuplet* t = note->chord()->tuplet();
       int actNotes = 1;
       int nrmNotes = 1;
-      int nrmTicks = 0;
+      //int nrmTicks = 0;
```
Comment (lasconic) [inline mscore/exportxml.cpp]: Please delete the lines.

Comment (Jojo-Schmitz) [reply mscore/exportxml.cpp]: done

**Comment by lasconic** [comment]
Thanks. Can you squash ? I will handle the backport to 2.2.

**Comment by Jojo-Schmitz** [comment]
Both commits can get applied to either branch, I just kept them separate just in case...
Squashed now

**Comment by lvinken** [comment]
Hi guys,

thanks for catching and fixing those issues. Xcode (obviously) did not warn me, so I was unaware.

Regards, Leon.

**Pull request merged**